### ISSUE TITLE
Augmentation that pads inputs to a fixed size

### ISSUE THREAD

**Issue by calebrob6** [kornia/kornia#1283]
## 🚀 Feature

A `kornia.augmentation.PadTo` module that pads inputs to a fixed size. 

## Motivation

Many datasets (e.g. ImageNet) are made up of images of different sizes. One way to standardize these for model training purposes is to resize / crop, however in some domains (e.g. remote sensing) resizing imagery is not desirable as it will change the spatial resolution of the image. In these cases, padding (zero, symmetric, etc.) can be used to standardize input sizes.  

## Pitch

A `kornia.augmentation.PadTo` module that uses `torch.nn.functional.pad` internally to pad inputs to a fixed size.

## Alternatives

We could implement this functionality in our own library, however we feel that it is general-purpose enough to belong in Kornia.

Some other context, `torchvision.transforms.Pad` exists, however this adds a fixed amount of padding to each input vs. padding each input to a fixed size.

## Additional context

We would like to use such functionality in `torchgeo`, a PyTorch domain library for geospatial data - https://github.com/microsoft/torchgeo.

@adamjstewart  @isaac

**Comment by edgarriba** [comment]
@calebrob6 thanks for opening this tiquet. I'll give a shot to that tomorrow :)

**Comment by calebrob6** [comment]
Thanks! 

This is what we came up with if it helps -- https://github.com/microsoft/torchgeo/blob/feature/padto_transform/torchgeo/transforms/transforms.py#L103. I'm happy to take a stab at this too if you run out of time!

**Comment by edgarriba** [comment]
I see - in principle you could implement similar to this one /cc @shijianjian https://github.com/kornia/kornia/blob/master/kornia/augmentation/augmentation.py#L47

**Comment by shijianjian** [comment]
@calebrob6 If you are using Kornia augmentations, I would suggest using [Augmentation Sequential](https://kornia.readthedocs.io/en/latest/applications/image_augmentations.html#best-practices-1-image-augmentation) that internally handles the mask property without any efforts.

**Comment by edgarriba** [comment]
@calebrob6 check this out https://github.com/kornia/kornia/pull/1286

We are going to merge during the weekend to be included next weeks in the release.

**Comment by calebrob6** [comment]
Awesome!!! thanks a lot!
